platform/v1alpha1/types.go

```diff
@@ -0,0 +1,86 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Package struct {
```

Comment:
Please add a godoc here, something along the lines of `Pacakage contains fields to configure which OLM package this PlatformOperator will install`